### DISCUSSION TITLE
Disable Captcha verification?

### DISCUSSION THREAD

**Discussion by ruijorgesilva1** [playsms/playsms#664] [Q&A] [answered]
Is it possible to disable Captcha verification?
I'm working on a private lan/home server and i'm the only user.
The problem is, in most of times, captcha is almost unreadable! Letter are small or lines are in front of letter and becames very difficult to read.
Thanks for the great work!
Considering giving donation

**Comment by Guerzizeb** [comment, marked as answer]
Yes, it's possible for ether login, reset password or registration.
The config file is under : web/plugin/core/auth/config.php

// enable/disable captcha
$auth_captcha_form_login = false;
$auth_captcha_form_forgot = false;
$auth_captcha_form_register = true;

**Reply by ruijorgesilva1** [reply]
Thank You Very much for your kind help!
